pkg/extract/redhat/csaf/csaf.go

```diff
@@ -231,19 +240,19 @@ type product struct {
 	repositories    []string
 }
 
-type ass struct {
-	advisory string
-	severity severity
-	status   status
+type ASS struct {
```

Comment:
There seems to be no need to export it?
```suggestion
type ass struct {
```

Reply:
done in a2dd727